src/richie/plugins/large_banner/forms.py

```diff
@@ -0,0 +1,30 @@
+"""
+Large banner plugin forms
+"""
+from django import forms
+
+from djangocms_text_ckeditor.widgets import TextEditorWidget
+
+from .models import LargeBanner
+
+
+class LargeBannerForm(forms.ModelForm):
+    """
+    Plugin form used to fill its content from frontend admin.
+    """
+
+    class Meta:
+        """
+        Form meta attributes
+        """
+
+        model = LargeBanner
+        widgets = {"content": TextEditorWidget}
```

Comment:
We shall have a specific CKEditor configuration for this type of field (which could easily break visual design)

Reply:
sorry: TextEditorWidget is good if this is raw text

Reply:
CKEditor won't have any real meaning if it's just raw plain text since CKEditor produces paragraph tags they would be escaped (since @sampaccoud have mentioned to remove `safe` filter).

By the way, i thing raw plain text would be useless in this plugin, in real world you never want to include Hero content without basic inline tags.

I propose to indeed make a "inline tags only" (bold, italic, link and paragraph) CKEditor config with every other features removed, also remove the "Source" panel so user won't be able to pass through our rules. Also, i will watch to have an optional setting to choose other config, third user could so use more tags if he wants to from its own richie project.

Reply:
But we can apply some markup via the linked template no?

Reply:
Yes but markup applied from template is just around inputed content, not inside.